source/docs/hardware/hardware-basics/robot-battery.rst

```diff
@@ -0,0 +1,188 @@
+
+.. image:: images/robot-battery/broken-terminal.png
+  :alt: One terminal of an FRC battery fully detached from the battery.
+
+Even if it doesn't get broken, the battery will not be able to provide the same amount of current if the :ref:`connectors are loose <docs/hardware/hardware-basics/preemptive-troubleshooting:Check Battery Connections>`.
+
+//TODO is this a good place to talk about locking down the SB50 so you don't pull on wire, or is that handled on Common troubleshooting?
+
+Dropping the batteries can bend the internal plates, create bulges, or in the worst case - crack the battery case open.
+While we do use AGM [absorbant glass mat] technology for safety, when a cell is punctured it still will leak some liquid.
+This is why FIRST recommends teams have a battery spill kit available. //TODO add link?
+
+Finally, certain older battery chargers can *overcharge* the battery, resulting in boiling off some of the battery acid - another reason for a spill kit.
+
+.. image:: images/robot-battery/boiled.jpg
+  :alt: An FRC battery that was boiled by a high current dumb charger.
+
+Most modern chargers can charge the battery indefinitely without additional harm.
+The picture above is from a high current charger with no maintenance mode that boiled the acid until it produced steam and high pressure that opened the safety vents.
+As such a modern smart charger is highly recommended.
+
+Damaged batteries should be safely disposed of as soon as possible. All retailers that sell large SLA batteries, like car batteries, should be able to dispose of it for you. They may charge a small fee, or provide a small "core charge refund", depending on your state law.
+
+.. warning:: **DO NOT** attempt to "repair" damaged or non-functional batteries.
+
+Battery Construction & Tools
+-----------------------------
+
+Battery Leads
+^^^^^^^^^^^^^^^^^^^
+
+Standard 6AWG, SB50 battery leads normally come in the Kit of Parts, courtesy of AndyMark. Teams are highly encouraged to use these leads.
+
+//TODO cite rules here
+The leads must be copper, minimum size 6AWG and maximum length 12" - there is no maximum size. Teams are recommended to use the standard 6AWG, it is sufficient for almost all robots.
+The Anderson SB Connector may be either a standard PINK or RED SB50, or another Anderson SB connector such as the SB120. Teams are *STRONGLY* recommended to use the standard RED or PINK SB50 for interoperability, as the other color housings will not intermate and you will be unable to borrow batteries, chargers, or other equipment.
+Terminals for the battery tabs are available online and through electrical supply houses, sold by the accepted wire size and the tab hole diameter, sometimes listed on catalogs as a "post diameter".
+
+Teams are strongly recommended to use either use properly rated tools and stringent quality control practices for the crimping process, or use vendor-made leads. //TODO do we talk about crimping anywhere that this should link to?
```

Comment:
I made an issue for a crimping article we should eventually link here but that can be part of that document.